frontend/src/metabase/palette/shortcuts/index.ts

```diff
@@ -0,0 +1,9 @@
+import { dashboardShortcuts } from "./dashboard";
+import { globalShortcuts } from "./global";
+// import { questionShortcuts } from "./question";
```

Comment:
None of these shortcuts have been implemented yet, so I've commented out their definitions